pkg/lorry/engines/models/replca_role_types.go

```diff
@@ -29,3 +34,20 @@ const (
 	LEARNER   = "Learner"
 	CANDIDATE = "Candidate"
 )
+
+func IsLeaderOrPrimaryOrMaster(role string) bool {
+	return role == LEADER || role == PRIMARY || role == MASTER
```

Comment:
Should use case insensitive string comparision?

Reply:
Consider using mgr.IsLeader instead of hard coding the role.